packages/@tinacms/webpack-helpers/package.json

```
@@ -0,0 +1,20 @@
+{
+  "name": "@tinacms/webpack-helpers",
+  "version": "0.0.0",
+  "license": "Apache-2.0",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/tinacms/tinacms.git",
+    "directory": "packages/@tinacms/webpack-alias"
+  },
+  "scripts": {
+    "watch": "tinacms-scripts watch",
+    "build": "tinacms-scripts build",
+    "dev": "tinacms-scripts dev"
```

These commands are unnecessary:

```suggestion
    "watch": "echo 'watch'",
    "build": "echo 'build'",
    "dev": "echo 'dev'"
```